### PR TITLE
[WIP] Bumped up python version

### LINKED ISSUE #33: Bump project to Python 3.11
Update the project to Python 3.11. Similar changes were done in Pretix here https://github.com/pretix/pretix/pull/2982/files

### PR DESCRIPTION
**Pull request by elle-Bee**
Fixes #33 

- Updated the workflow matrix in `tests.yml` to use `[3.7, 3.10, 3.11]` instead of `[3.6, 3.7, 3.8]`.

- Updated PostgreSQL version `9.5` -> `11` in `/doc/admin/installation`

### CODE REVIEW

**Review by NaviRocker — APPROVED** [review]
Good to Go!

**Comment by hongquan** [comment]
Please remove support for v3.7 also. We will support v3.10 onward.
The tactics is to support the Python included in the latest Ubuntu LTS. At the time of writing this, latest Ubuntu LTS is Ubuntu 22.04, and it comes with Python v3.10.

File: .github/workflows/tests.yml [hunk 1]
```diff
@@ -18,22 +18,22 @@ jobs:
     name: Tests
     strategy:
       matrix:
-        python-version: [3.6, 3.7, 3.8]
+        python-version: [3.7, 3.10, 3.11]
```
Comment (hongquan) [inline .github/workflows/tests.yml]: Please remove 3.7.

**Review by hongquan — CHANGES_REQUESTED** [review]

File: .github/workflows/tests.yml (hunk 1 above)
Comment (elle-Bee) [reply .github/workflows/tests.yml]: Made the changes now it only refers to versions `3.10` and `3.11`

**Comment by elle-Bee** [comment]
Made the changes now it only refers to versions `3.10` and `3.11`

**Comment by elle-Bee** [comment]
hey @hongquan any updates?

**Comment by hongquan** [comment]
I am sorry, @elle-Bee. Because the current codebase of `eventyay-tickets` is depending on very old libraries, I'm not sure if they are really compatible with Python v3.10+, especially libraries which are written in C/C++. So I have to hold your PR in pending until another task, which is to upgrade Django, finishes. The other task will involve upgrading dependencies as well, and help us test this PR easier.

**Comment by hongquan** [comment]
Could you work on another task to remove Python 3.6, 3.7 from GitHub Action script?

**Comment by hongquan** [comment]
Please resolve conflict @elle-Bee

**Comment by elle-Bee** [comment]
resolved the conflicts with the dockerfile

**Comment by elle-Bee** [comment]
hey @hongquan any updates?

**Comment by mariobehling** [comment]
The Python upgrade can only be done as part of the Django system upgrade. The build does not work with the updates in this issue currently.

**Comment by elle-Bee** [comment]
oh okay

**Comment by mariobehling** [comment]
We have successfully progressed with the Django upgrade. Please make your PR against the dev branch now.

**Comment by mariobehling** [comment]
This has been solved in the meantime. Therefore closing it.

**Pull request closed without merging**